diff --git a/base/BUILD.gn b/base/BUILD.gn
index 08ffb229388bc2deb265901de1b7868eb233fc96..f4788ff894aef614a2b0731517af71cc2cb2ef15 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4817,6 +4817,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5473,6 +5474,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
