diff --git a/chrome/browser/ui/chrome_pages.cc b/chrome/browser/ui/chrome_pages.cc
index 94e86a92f7314407ecff94b41d27fcb935c18254..d3b78191316749787d8e2c0b6667bb7c176de7c3 100644
--- a/chrome/browser/ui/chrome_pages.cc
+++ b/chrome/browser/ui/chrome_pages.cc
@@ -436,7 +436,7 @@ void ShowSlow(Browser* browser) {
 #endif
 }
 
-GURL GetSettingsUrl(std::string_view sub_page) {
+BRAVE_GET_SETTINGS_URL
   return GURL(base::StrCat({kChromeUISettingsURL, sub_page}));
 }
 
