diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 78b94ee0de3800ef75177cbf9c6f2ddf61c42be7..e35dbfde71bcf83b73f8cb119f8eacc5188c077e 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -5166,6 +5166,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
