diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 85b5fd39dfb5ab4fc980b427321420a9c3aac25f..355fa117acf9d981cc91de0831c2bc7e6839b8d3 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -404,6 +404,7 @@ void ToolbarView::Init() {
     media_button_ = AddChildView(std::move(media_button));
   }
 
+  BRAVE_TOOLBAR_VIEW_INIT
   avatar_ = AddChildView(std::make_unique<AvatarToolbarButton>(browser_view_));
   bool show_avatar_toolbar_button = true;
 #if BUILDFLAG(IS_CHROMEOS)
