diff --git a/chrome/test/data/webui/settings/BUILD.gn b/chrome/test/data/webui/settings/BUILD.gn
index 03cce5ea7bf379a05ff701b47554fc13702e254f..2197fe40540cc04d6f5f7f672e4234c5e02050ad 100644
--- a/chrome/test/data/webui/settings/BUILD.gn
+++ b/chrome/test/data/webui/settings/BUILD.gn
@@ -291,6 +291,7 @@ build_webui_tests("build") {
     ts_definitions +=
         [ "//tools/typescript/definitions/quick_unlock_private.d.ts" ]
   }
+  import("//brave/browser/resources/settings/sources.gni") ts_definitions += brave_settings_ts_definitions
 
   ts_deps = [
     "//chrome/browser/resources/settings:build_ts",
