diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index 2cefce34690b7f0ee4e79c3c14101148301b0612..7658d6659f1920900f289dc394e40512fbc26ba6 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -141,6 +141,7 @@ source_set("internal") {
       deps += [ "//chrome/browser/ui:overscroll_pref_manager" ]
     }
   }
+  import("//brave/browser/ui/browser_window/internal/sources.gni") deps += brave_browser_window_internal_deps
 }
 
 source_set("unit_tests") {
