diff --git a/components/os_crypt/common/keychain_password_mac.mm b/components/os_crypt/common/keychain_password_mac.mm
index 0206d808c72b38e3d3abc98bd9a44fc47ed6c4da..f4bb9b720d511e3516a9e40fcac29a9bb00b80ea 100644
--- a/components/os_crypt/common/keychain_password_mac.mm
+++ b/components/os_crypt/common/keychain_password_mac.mm
@@ -110,12 +110,14 @@ base::expected<std::string, OSStatus> GetPasswordImpl(
 
 // static
 KeychainPassword::KeychainNameType& KeychainPassword::GetServiceName() {
+  BRAVE_GET_SERVICE_NAME
   static KeychainNameContainerType service_name(kDefaultServiceName);
   return *service_name;
 }
 
 // static
 KeychainPassword::KeychainNameType& KeychainPassword::GetAccountName() {
+  BRAVE_GET_ACCOUNT_NAME
   static KeychainNameContainerType account_name(kDefaultAccountName);
   return *account_name;
 }
