diff --git a/components/omnibox/composebox/BUILD.gn b/components/omnibox/composebox/BUILD.gn
index 51f263346eaa67785c312babd09681883eb01a93..8f61bd9079149dffbfde4946b8160eddf4e33544 100644
--- a/components/omnibox/composebox/BUILD.gn
+++ b/components/omnibox/composebox/BUILD.gn
@@ -5,6 +5,7 @@
 import("//mojo/public/tools/bindings/mojom.gni")
 
 mojom("mojo_bindings") {
+  generate_legacy_js_bindings = true
   sources = [ "composebox_query.mojom" ]
   public_deps = [ "//mojo/public/mojom/base" ]
 
