diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 21897501f47d1eed057c1d2ac6e163ff82b596c4..b62874202cd2c6bd35b607189f526310821507ee 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3708,6 +3708,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
