diff --git a/services/network/cors/cors_url_loader.cc b/services/network/cors/cors_url_loader.cc
index 5139ae0fcf77d0eca984e64e12c33adff8cfc7ae..b38ed02dc237575f2f46ede0e1fe2e3449c0f9c3 100644
--- a/services/network/cors/cors_url_loader.cc
+++ b/services/network/cors/cors_url_loader.cc
@@ -878,6 +878,7 @@ void CorsURLLoader::StartRequest() {
   };
 
   if (should_include_origin_header()) {
+    BRAVE_CORS_URL_LOADER_START_REQUEST
     if (tainted_) {
       request_.headers.SetHeader(net::HttpRequestHeaders::kOrigin,
                                  url::Origin().Serialize());
