diff --git a/ui/views/view.h b/ui/views/view.h
index 92732d1ed1332db8967839fe808954bf6f5ed1e8..be1d0e34b8e253921f73dd5c6914308a5ce100ac 100644
--- a/ui/views/view.h
+++ b/ui/views/view.h
@@ -327,6 +327,7 @@ class VIEWS_EXPORT View : public ui::LayerDelegate,
     FRIEND_TEST_ALL_PREFIXES(WebViewUnitTest, CrashedOverlayView);
 
     OwnedByClientPassKey() = default;
+    BRAVE_VIEW_OWNED_BY_CLIENT_PASS_KEY
   };
 
   using PassKey = base::NonCopyablePassKey<View>;
