diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 375efd4bc026340019d1ae0e5445bf233ff53f4a..bc3f57f0cbd902eee9451613f73c4bab6bd4c118 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2737,6 +2737,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
